Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -730,6 +751,16 @@ void ABenchmarkGymGameModeBase::ReadWorkerFlagValues(USpatialWorkerFlags* Spatia
 		SetLifetime(FCString::Atoi(*LifetimeString));
 	}
 
+	if (SpatialWorkerFlags->GetWorkerFlag(CubeRespawnBaseTimeWorkerFlag, CubeRespawnBaseTimeString))
+	{
+		CubeRespawnBaseTime = FCString::Atof(*CubeRespawnBaseTimeString);
```

Comment:
This returns 0.0 on failure to make any sense of the string, is that acceptable?
(It calls `_tstof`  -> `_wtof` under the hood https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/atof-atof-l-wtof-wtof-l?view=msvc-170).

Reply:
Would actually suggest wrapping this in 
```
if (CubeRespawnBaseTimeString.IsNumeric())
{
}
// or
checkf(CubeRespawnBaseTimeString.IsNumeric(), TEXT("Not a valid number"));
```

Reply:
Might even be worth wrapping into a free function since there is a lot of string to float in this commit.

Reply:
Yeah it would be invalid to pass it anything but a number. We also output the values set by these calls in a general log at the bottom of this function, so the user could debug if it's a problem.

Reply:
But it won't be an error, could be difficult to spot if we are just leaning on printing output values?

Reply:
I think this is fine. If there was an issue with the scenario, first thing you'd check is the settings.

Reply:
ok agree to disagree, not a fan of silent errors, you have more context so consider it resolved : ).